tests/Integration/Command/ImportScorerCommandTest.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * Human Scoring Software
+ *
+ * @author antoinep@taotesting.com
+ * @license See LICENCE.md
+ */
+namespace App\tests\Integration\Command;
+
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class ImportScorerCommandTest extends KernelTestCase
+{
+    public function testExecute()
+    {
+        $kernel = static::createKernel();
+        $application = new Application($kernel);
+        $command = $application->find('app:import-scorer');
+        $commandTester = new CommandTester($command);
+        $result = $commandTester->execute(
+            [
+                'path' => __DIR__ . '/../../Samples/scorer-test.csv',
+            ],
+            [
+                'capture_stderr_separately' => true
+            ]
+        );
+        $this->assertEquals(0, $result);
+    }
+
+    public function testPathMissing()
+    {
+        $kernel = static::createKernel();
+        $application = new Application($kernel);
+        $command = $application->find('app:import-scorer');
+        $commandTester = new CommandTester($command);
+        $result = $commandTester->execute(
+            [
+                'path' => null,
+            ],
+            [
+                'capture_stderr_separately' => true
+            ]
+        );
+        $this->assertEquals(1, $result);
+    }
+
+    public function testWithWrongPath()
+    {
+        $kernel = static::createKernel();
+        $application = new Application($kernel);
+        $command = $application->find('app:import-scorer');
+        $commandTester = new CommandTester($command);
+        $result = $commandTester->execute(
+            [
+                'path' => __DIR__ . '/../../Samples/wrongPath.csv',
+            ],
+            [
+                'capture_stderr_separately' => true
+            ]
+        );
+        $this->assertEquals(1, $result);
```

Comment:
On each of the failure tests, you might also want to ensure that no data has been modified in your database.